pkg/api/job_runs.go

```diff
@@ -272,6 +436,13 @@ func runJobRunAnalysis(jobRun *models.ProwJobRun, compareRelease string, jobRunT
 		response.OverallRisk.Reasons = append(response.OverallRisk.Reasons,
 			fmt.Sprintf("%d tests failed in this run: High", len(jobRun.Tests)))
 		return response, nil
+
+	// Return early is this is for a never-stable job
+	case neverStableJob:
+		response.OverallRisk.Level = apitype.FailureRiskLevelLow
+		response.OverallRisk.Reasons = append(response.OverallRisk.Reasons,
+			"Job is marked as never-stable")
+		return response, nil
```

Comment:
Is it feasible to proceed with test analysis on these so we could be watching risk of jobs we're hoping to get OUT of never-stable, or is that largely impossible? With variant matching probably not feasible, but with job name we might get something useful?

Reply:
We can try.  If it looks to be at all flaky I say dump it and get value from the other 99% though.